src/shared/analytics/shared/getTotalWalletsBalance.ts

```diff
@@ -1,7 +1,13 @@
 import type { Portfolio } from 'defi-sdk';
 import { client } from 'defi-sdk';
 
-export async function getAddressesPortfolio(addresses: string[]) {
+export async function getAddressesPortfolio({
```

Comment:
Let's move this function out of analytics folder

Reply:
Since now I'm using the `useAddressPortfolio` from defi-sdk, I'll revert my changes to this function and leave it as it was before